docs/windows-broker.md

```diff
@@ -0,0 +1,100 @@
+
+If your machine isn't Intune/Azure AD-joined, or is Intune/Azure AD-joined to a different tenant, WAM will present you with the following dialog box:
+
+![Consent dialog](img/aad-questions.png)
+
+Depending on what you click, one of three things can happen:
+
+- If you leave "allow my organization to manage my device" checked and click "OK", your computer will be registered with the Azure AD tenant backing the organization.
+It may also be Intune-enrolled, meaning an administrator can deploy policies to your machine: requiring certain kinds of sign-in, turning on antivirus and firewall software, and enabling BitLocker.
+Your identity will also be available to other apps on the computer for signing in, some of which may do so automatically.
+![Example of policies pushed to an Intune-enrolled device](img/aad-bitlocker.png)
+- If you uncheck "allow my organization to manage my device" and click "OK", your computer won't be registered with Azure AD or Intune-enrolled.
+Your identity will be available to other apps on the computer for signing in.
+Other apps may log you in automatically or prompt you again to allow your organization to manage your device.
+- If you instead click "No, sign in to this app only", your machine will not be Intune-enrolled, so no policies can be enforced, and your identity won't be made available to other apps on the computer.
+
+#### Removing device management
+If you've allowed your computer to be managed and want to undo it, you can go into **Settings**, **Accounts**, **Access work or school**.
+In the section where you see your email address and organization name, click **Disconnect**.
+
+![Finding your work or school account](img/aad-work-school.png)
+
+![Disconnecting from Azure AD](img/aad-disconnect.png)
+
+### For Microsoft accounts
+When you sign into an Azure DevOps organization backed by Microsoft account (MSA) identities (email addresses like `@outlook.com` or `@gmail.com` fall into this category), you may be prompted to select an existing "work or school account" or use a different one.
+
+In order to sign in with an MSA you should continue and select "Use a different [work or school] account", but enter your MSA credentials when prompted.
+This is due to a configuration outside of our control.
+We expect this experience to improve over time and a "personal account" option to be presented in the future.
+
+![Initial dialog to choose an existing or different account](img/get-signed-in.png)
+
+If you've connected your MSA to Windows or signed-in to other Microsoft applications such as Office, then you may see this account listed in the authentication prompts when using GCM.
+For any connected MSA, you can control whether or not the account is available to other Microsoft applications in **Settings**, **Accounts**, **Emails & accounts**:
+
+![Allow all Microsoft apps to access your identity](img/all-microsoft.png)
+![Microsoft apps must ask to access your identity](img/apps-must-ask.png)
```

Comment:
Stick these in a table to get them side-by-side (with appropriate captions) so the layout is a little better (no more GIANT images)?

Reply:
I added a newline between them so they stack instead. Thanks!